sources/src/main/java/com/google/solutions/jitaccess/web/RuntimeConfiguration.java

```diff
@@ -59,7 +59,11 @@ public RuntimeConfiguration(Function<String, String> readSetting) {
     this.justificationHint = new StringSetting(
       List.of("JUSTIFICATION_HINT"),
       "Bug or case number");
-
+    
+    //
+    // Backend service id
+    //
+    this.backendServiceId = new StringSetting(List.of("BACKEND_SERVICE_ID"), null);
```

Comment:
(optional): Maybe prefix this with `IAP_`?

Reply:
doone